### PR TITLE
Added OnChange event for TextArea Kit

### PR DESCRIPTION
**Pull request by jasperfurniss**

### CODE REVIEW

File: app/pb_kits/playbook/pb_textarea/_textarea.jsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ type TextareaProps = {
   name?: String,
   rows?: Number,
   dark?: Boolean,
+  onChange?: Func
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_textarea/_textarea.jsx]:
```suggestion
  onChange?: InputCallback
```

Comment (thestephenmarshall) [reply app/pb_kits/playbook/pb_textarea/_textarea.jsx]: Add 

```jsx
import { InputCallback } from '../../types'
```

to get this type

File: app/pb_kits/playbook/pb_textarea/_textarea.jsx [hunk 2]
```diff
@@ -45,7 +47,7 @@ const Textarea = ( props: TextareaProps) => {
       <If condition={children}>
         {children}
       <Else/>
-        <textarea className={textarea_input} name={name} placeholder={placeholder} rows={rows} value={value} />
+        <textarea onChange={onChange} className={textarea_input} name={name} placeholder={placeholder} rows={rows} value={value} />
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_textarea/_textarea.jsx]:
```suggestion
        <textarea
            onChange={onChange}
            className={textarea_input}
            name={name}
            placeholder={placeholder}
            rows={rows}
            value={value}
        />
```

Comment (thestephenmarshall) [reply app/pb_kits/playbook/pb_textarea/_textarea.jsx]: for legibility

File: CHANGELOG.md [hunk 3]
```diff
@@ -53,6 +53,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Added an OnChange event handler for the TextArea React Kit ([#505][] @jasperfurniss)
```
Comment (thestephenmarshall) [inline CHANGELOG.md]: 👍

Comment (rafbgarcia) [reply CHANGELOG.md]: Nice! Please create a `### Changed` category under `[Unreleased]` section and move it there?


```
### Changed 

- Added an OnChange event handler for the TextArea React Kit ([#505][] @jasperfurniss)

[#505]: https://github.com/powerhome/playbook/pull/505

**Pull request merged**